docs/feature-debugging.md

```diff
@@ -0,0 +1,83 @@
+A powerful features of the SAFE Stack is that it supports combined client / server debugging in F#. This means that you can use a single IDE (in this case, [Visual Studio Code](https://code.visualstudio.com/)) to debug *both client and server code* in the **same IDE session**.
+
+As of v0.38 of the SAFE Template, new SAFE applications come pre-configured with build, launch and debugging support in VS Code.
+
+## Prerequisites
+1. **Install [Google Chrome](https://www.google.com/chrome/)**: Enables for client-side debugging.
+1. **Install the [Redux Dev Tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en)** extension for Chrome: Provides improved debugging support in Chrome with Elmish.
+1. **Configure VS Code** with the following extensions:
+    * [Ionide](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp): Provides F# support to Code.
+    * [C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp): Provides .NET Core debugging support.
+    * [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome): Provides integrated client-side debugging in Code.
+
+## Launching and debugging a SAFE application in VS Code
+The SAFE template allows you to launch a SAFE Stack application with both client and server debugging enabled automatically.
+
+1. Create a new SAFE application using the [SAFE template](template-overview)
+1. Open VS Code in the folder containing your application
+1. Go to the Debug pane (`CTRL + Shift + D`)
+ 
+    <center><img src="../img/feature-debugging-1.png" style="height: 400px;"/></center>
```

Comment:
probably not that important, but I think the default VS Code configuration comes with the Activity Bar at left hand side